src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -177,19 +174,26 @@ typedef bmpl::vector<
     shape<UsedParticleShape>,
     interpolation<UsedField2Particle>,
     current<UsedParticleCurrentSolver>,
-    massRatio<MassRatioIons>,
-    chargeRatio<ChargeRatioIons>
-> ParticleFlagsIons;
+    massRatio<MassRatioElectrons>,
+    chargeRatio<ChargeRatioElectrons>
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
+    ,synchrotronPhotons<PIC_Photons>
+#endif
+#if(ENABLE_BREMSSTRAHLUNG == 1)
```

Comment:
@ComputationalRadiationPhysics/picongpu-maintainers 

Should we keep the `ENABLE_BREMSSTRAHLUNG` in the gernic picongpu part or should we only show this as  within the `LaserWakefield` example.
The reason why I am asking is that the user need to define `ENABLE_BREMSSTRAHLUNG` within the cmake to enable this code path.hHow to enable is only visiable in the example `LaserWakefield` in the file [cmakeFlags](https://github.com/ComputationalRadiationPhysics/picongpu/pull/1504/files#diff-17e35932abac366d12163ba8d613a304L42)